calcom/cal.com
# Hide attendees from calendar description

## What does this PR do?



This PR hides seats attendees from the description of calendar events, while allowing the organizer to see the attendees in the calendar event.

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Connect a GCal
- Create an event type with seats
- Book at least two attendees in the slot
- The body of the event should show only the organizer but on the event it's self the attendees should be everyone.

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
